serverlessworkflow/modules/ROOT/pages/integrations/custom-functions-knative.adoc

```diff
@@ -110,7 +110,7 @@ Save the Knative service name (`custom-function-knative-service`) to use it in t
 [source,bash]
 ----
 curl -X 'POST' \
-  '<URL>/knativeFunction' \ <1>
+  '<URL>/plainJsonKnativeFunction' \ <1>
```

Comment:
I think it's worth expanding this section to add CE and a link back to the custom functions support. One paragraph should do.